src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -653,10 +655,8 @@ public Object onTraversalSuccess() {
                     if (goodModel.failedToConverge) {
                         if (outputModel != null) {
                             final GATKReport report = writeModelReport(goodModel, null, USE_ANNOTATIONS);
-                            try (final PrintStream modelReportStream = new PrintStream(outputModel)) {
+                            try (final PrintStream modelReportStream = new PrintStream(outputModel.getOutputStream())) {
                                 report.print(modelReportStream);
-                            } catch (FileNotFoundException e) {
-                                throw new UserException.CouldNotCreateOutputFile("File: (" + outputModel + ")", e);
```

Comment:
Handle errors on the `PrintStream` (by `checkError()` if that's the only way...)

Reply:
Here I added a `checkError` call. `GATKReport` still has(/had) issues, but thats in the uri ticket.